### DISCUSSION TITLE
exec ./honeygain: exec format error

### DISCUSSION THREAD

**Discussion by Raffy142** [engageub/InternetIncome#154] [Q&A]
Hi, today i have this problem
i use ubuntu 22.04 and when i restart this problem appears in the logs
![image](https://github.com/user-attachments/assets/88d04764-92cf-400d-9b60-34bca9c983c8)
![Screenshot_2](https://github.com/user-attachments/assets/37ec2334-0453-4a9f-bf26-1adde6bde953)

**Comment by engageub** [comment]
Hello,
It looks like you are using arm64 architecture.
Older version of honeygain is being used with proxies as it has been confirmed to be working fine.
It only supports amd64 architecture by default.
You may change the version to the latest in the code by replacing `honeygain/honeygain:0.6.6` with `honeygain/honeygain` in `internetIncome.sh` file.
If your account works fine with arm64 architecture, please update here.

Thank you
